test/whitelisted_crowdsale.js

```diff
@@ -73,3 +76,77 @@ contract('WhitelistedCrowdsale', async (accounts) => {
   });
 
 });
+
+contract('WhitelistedCrowdsale', async (accounts) => {
+  let owner = accounts[0];
+  let holder = accounts[1];
+  let escrow_account = accounts[2];
+  let user = accounts[3];
+  let userBalance = web3.eth.getBalance(user);
+  let defaultRateForPrivateOffer = 14000;
+
+  let controller = null;
+  let instanceWhitelistedCrowdsale = null;
+  let token = null;
+
+  beforeEach("setup contract for each test", async () => {
```

Comment:
In cases, you do, it is not needed to redeploy contracts before each test.
Also, in case you need to redeploy, you can just type
contract (...) and so on